IFC4x3/Properties/t/Transmissivity_1md2GJC9f2MflqkMhIDr4K/Documentation.md

```diff
@@ -0,0 +1 @@
+Parameter governing the flow of water flowing per unit width of the saturated zone of a continuous aquifer (measured in a direction orthogonal to that of flow), and per unit of hydraulic gradient.
```

Comment:
Not clear what units of measurement there are.

Reply:
Reused Transmissivity from PSet_InSituTestHydro.